File: mods/sable/common/src/main/java/dev/ryanhcode/sable/mixinterface/udp/ServerConnectionListenerExtension.java
package dev.ryanhcode.sable.mixinterface.udp;

import dev.ryanhcode.sable.network.udp.SableUDPServer;
import io.netty.channel.Channel;

public interface ServerConnectionListenerExtension {
    void sable$setupUDPServer(Channel channel);

    SableUDPServer sable$getServer();
}

File: src/main/kotlin/dev/codinglabs/modpack/mixin_interfaces/ServerConnectionListenerExtension.kt
package dev.codinglabs.modpack.mixin_interfaces

import dev.codinglabs.modpack.rapier_entity.network.UdpServer
import io.netty.channel.Channel

interface ServerConnectionListenerExtension {
    fun `codinglab$setupUdpServer`(channel: Channel)

    val `codinglab$server`: UdpServer
}
